perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -152,8 +221,10 @@ def Cleanup(benchmark_spec):
     benchmark_spec: The benchmark specification. Contains all data that is
         required to run the benchmark.
   """
-  vms = benchmark_spec.vms
-  vm = vms[0]
+  vm = benchmark_spec.vms[0]
   logging.info('FIO Cleanup up on %s', vm)
-  vm.RemoveFile(flags.FLAGS.fio_jobfile)
-  vm.RemoveFile(vm.GetScratchDir() + flags.FLAGS.fio_benchmark_filename)
+  vm.RemoveFile(REMOTE_JOB_FILE_PATH)
+  if not FLAGS.against_device and not FLAGS.fio_jobfile:
+    # If the user supplies their own job file, then they have to clean
+    # up after themselves, because we don't know their temp file name.
+    vm.RemoveFile(vm.GetScratchDir() + 'fio-temp-file')
```

Comment:
I am confused here, why we don't know their temp file name. PKB upload the job file to remote vm from data/ directory?